### PR TITLE
Update/altv

### PR DESCRIPTION
**Pull request by paz**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### Changes to an existing Egg:

1. [x] Have you added an explanation of what your changes do and why you'd like us to include them?
2. [x] Have you tested your Egg changes?

In summary, this adds nodejs to the image and simplifies the install script to use the updater script mentioned in the installation page on the altV wiki

### CODE REVIEW

File: gta/altv/egg-alt--v.json [hunk 1]
```diff
@@ -1,40 +1,45 @@
...
     "author": "parker@parkervcp.com",
     "description": "alt:V Multiplayer a third-party multiplayer modification for Grand Theft Auto: V",
-    "image": "quay.io\/parkervcp\/pterodactyl-images:debian_dotnet",
-    "startup": "sleep 2 && .\/altv-server",
+    "features": null,
+    "images": [
+        "quay.io\/parkervcp\/pterodactyl-images:game_altv"
+    ],
+    "file_denylist": "",
+    "startup": ".\/update.sh && chmod +x .\/altv-server && .\/altv-server",
```
Comment (parkervcp) [inline gta/altv/egg-alt--v.json:16]: no scripts in the startup.

File: gta/altv/egg-alt--v.json [hunk 2]
```diff
@@ -1,40 +1,45 @@
...
+            "script": "#!\/bin\/bash\r\n# AltV Install Script\r\n#\r\n# Server Files: \/mnt\/server\r\n# BUILD=stable ## this should be an egg variable.\r\napt update && apt install -y libatomic1 zip unzip jq curl wget libc-bin && cd \/mnt\/server && wget -O 'update.sh' 'https:\/\/raw.githubusercontent.com\/Lhoerion\/altv-serverupdater\/master\/update.sh' && chmod +x .\/update.sh && .\/update.sh && chmod +x .\/altv-server\r\necho -e \"install complete\"",
```
Comment (parkervcp) [inline gta/altv/egg-alt--v.json:25]: I will not accept you pulling another script to run the install. Users can run reinstall in 1.2+ so they can update that way.

**Review by parkervcp — CHANGES_REQUESTED** [review]

**Comment by paz** [comment]
here, i don't think the script will change much anyway. I hope this is ok

**Comment by paz** [comment]
oops i forgot to change startup command, hold on
